ida/ida_import.py

```diff
@@ -82,6 +83,8 @@ def import_function_comments(comments, sections):
 def import_line_comments(comments, sections):
     for addr, comment in comments.items():
         addr = adjust_addr(sections, int(addr))
+        if addr is None:
+            continue
```

Comment:
I ran into errors when addr was None while importing into IDA, so I added a continue here like the other functions.